packages/invoices/src/OutboundShipment/DetailView/Footer.tsx

```diff
@@ -52,79 +52,81 @@ export const Footer: FC<OutboundDetailFooterProps> = ({ draft, save }) => {
   return (
     <AppFooterPortal
       Content={
-        <Box
-          gap={2}
-          display="flex"
-          flexDirection="row"
-          alignItems="center"
-          height={64}
-        >
-          <ToggleButton
-            disabled={!isInvoiceEditable(draft)}
-            value={!!draft.onHold}
-            selected={!!draft.onHold}
-            onClick={(_, value) => {
-              draft.update?.('hold', !value);
-            }}
-            labelKey="label.hold"
-          />
-
-          <StatusCrumbs
-            statuses={outboundStatuses}
-            statusLog={createStatusLog(draft)}
-            statusFormatter={getStatusTranslation}
-          />
+        draft && (
```

Comment:
hmm I guess I dislike the optional chaining

Reply:
nothing else really changed here - just the calls for statuses as the draft is now definitely a draft.. 🙄

Reply:
fair enough! that was a lot of chaining